src/torrent_info.cpp

```diff
@@ -473,7 +473,7 @@ namespace {
 			return false;
 		}
 
-		std::time_t const mtime = std::time_t(dict.dict_find_int_value("mtime", 0));
+		auto const mtime = std::time_t(dict.dict_find_int_value("mtime", 0));
```

Comment:
it would be even shorter to remove the `std::time_t` on the right hand side, like this:
`std::time_t const mtime(dict.dict_find_int_value("mtime", 0));`
I'm not sure which one I prefer